ISSUE_TEMPLATE.md

```diff
@@ -3,3 +3,4 @@
 * * * *
 
 - [ ] I searched or browsed the repo’s other issues to ensure this is not a duplicate.
+- [ ] I am an instance admin who has checked that I'm running a [tagged release](https://github.com/tootsuite/mastodon/releases) and _not_ the latest master.
```

Comment:
we also want users to be able to contribute tho... I'd word it more like 

[ ] This bug happens on a [tagged release](https://github.com/tootsuite/mastodon/releases) and not on `master` (If you're a user, don't worry about this)